reference/conanfile/methods.rst

```diff
@@ -107,7 +107,7 @@ The syntax of copy is as follows:
 
 .. code-block:: python
 
-   self.copy(pattern, dst, src, keep_path=False, symlinks=None, excludes=None)
+   self.copy(pattern, dst, src, keep_path=True, links=False, symlinks=None, excludes=None, ignore_case=False)
```

Comment:
Actually the ``links`` argument is kept in code for backwards compatibility, but it is confusing from the UI. If symlinks is defined, links will take its value. Maybe it could be removed already, but I am always afraid to break users.